tests/integration/docusaurus/tutorials/quickstart/v1_sql_quickstart.py

```diff
@@ -0,0 +1,56 @@
+"""
+This guide demonstrates how to use Great Expectations in a quick-start
+notebook environment with SQL. It is useful to explore and understand how Great Expectations
+works, using your own data.
+
+Note:
+- Do not follow this workflow when you are building a durable workflow because it uses
+ephemeral assets.
+- Do not use this workflow for embedding in a pipeline or CI system, because it uses an
+iterative process for trying and refining expectations.
+"""
+
+# <snippet name="tests/integration/docusaurus/tutorials/quickstart/v1_sql_quickstart.py import_gx">
+import great_expectations as gx
+import great_expectations.expectations as gxe
+
+# </snippet>
+
+# Set up
+# NOTE: Context is a singleton now. Once the context has been set instantiated in a session
+# <snippet name="tests/integration/docusaurus/tutorials/quickstart/v1_sql_quickstart.py get_context">
+context = gx.get_context()
+# </snippet>
+
+connection_string = "postgresql://postgres:postgres@localhost:5432/postgres"
+# <snippet name="tests/integration/docusaurus/tutorials/quickstart/v1_sql_quickstart.py connect_to_data">
+batch = context.sources.pandas_default.read_sql(
+    "SELECT * FROM yellow_tripdata_sample_2019_01", connection_string
+)
+# </snippet>
+
+# <snippet name="tests/integration/docusaurus/tutorials/quickstart/v1_sql_quickstart.py create_expectation">
+expectation = gxe.ExpectColumnValuesToNotBeNull(
+    column="pickup_datetime",
```

Comment:
Minor changes from draft PR defining these workflows:
- Explicit `column` kwarg instead of positional
- "pickup_datetime" instead of "pu_datetime"

Reply:
on the column name change -- I think we should update these to use newer than 2019 data so that we have the new column names if htat's what's going on.